src/content/app/genome-browser/components/browser-sidebar-modal/modal-views/navigate-modal/NavigateModal.scss

```diff
@@ -0,0 +1,102 @@
+@import 'src/styles/common';
+
+.modalWrapper {
+  background: $light-grey;
+}
+
+.navigateModal {
+  width: 272px;
+}
+
+.browserNavBarControls {
+  margin: 15px 0 0 40px;
+
+  .moveLeftButton,
+  .moveRightButton,
+  .zoomInButton,
+  .zoomOutButton {
+    display: inline-block;
+    margin-right: 35px;
+  }
+
+  .moveLeftButton,
+  .moveRightButton {
+    height: 22px;
+    width: 22px;
+  }
+
+  .zoomInButton,
+  .zoomOutButton {
+    height: 16px;
+    width: 16px;
+  }
```

Comment:
Shouldn't we do for the move and zoom buttons the same as you did for the browser reset button? Your BrowserNavButton defines its own width and height.